unix_integration/resolver/src/idprovider/system.rs

```diff
@@ -8,6 +8,8 @@ use kanidm_unix_common::unix_passwd::{CryptPw, EtcGroup, EtcShadow, EtcUser};
 use kanidm_unix_common::unix_proto::PamAuthRequest;
 use kanidm_unix_common::unix_proto::{NssGroup, NssUser};
 
+const SYSTEM_GID_BOUNDARY: u32 = 1000;
```

Comment:
Just so it's clear without having to dig into logic

```suggestion
// The minimum GID that Kanidm will consider for creating a UPG
const SYSTEM_GID_BOUNDARY: u32 = 1000;
```